public/js/menu.js

```diff
@@ -1,66 +1,67 @@
-window.onload = () => {
```

Comment:
`window.onload` can be dropped when using `<script defer>` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#defer